src/Controllers/Pages/EditApplicationController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Portal\Controllers\Pages;
+
+use Portal\Controllers\Controller;
+use Portal\Entities\Application;
+use Portal\Models\CohortsModel;
+use Portal\Services\AuthService;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Portal\Models\ApplicantsModel;
+use Slim\Views\PhpRenderer;
+
+class EditApplicationController extends Controller
+{
+    private $model;
+    private $view;
+    private ApplicantsModel $applicantsModel;
+    private CohortsModel $cohortsModel;
+    private AuthService $authService;
+
+
+    public function __construct(ApplicantsModel $model,
+                                PhpRenderer $view,
+                                AuthService $authService,
+                                ApplicantsModel $applicantsModel,
+                                CohortsModel $cohortsModel
```

Comment:
Pretty sure the rest of the application is formatted this way?

```suggestion
    public function __construct(
        ApplicantsModel $model,
        PhpRenderer $view,
        AuthService $authService,
        ApplicantsModel $applicantsModel,
        CohortsModel $cohortsModel
```

Reply:
Done